bin/install.sh

```diff
@@ -87,5 +87,8 @@ echo "Installation completed."
 
 read -p "You need to reboot the system for the installation to complete. Would you like to reboot now? (y/N)" -n 1 -r -s REBOOT && echo
 if [ "$REBOOT" == 'y' ]; then
+  if test -f ~/.screenly/wifi_set; then  rm ~/.screenly/wifi_set; fi
```

Comment:
Why do you wrap this into an if statement if it is repeated regardless of the outcome below?

Reply:
because the next line reboots the system

Reply:
Sure, but if you move up `if test -f ~/.screenly/wifi_set; then  rm ~/.screenly/wifi_set; fi` to before 88, it will have the same effect but you won't have to repeat it.

Reply:
in fact, there is a slight difference. it may take a couple of minutes before you hit "y" or "N".
but I think you're right. if the viewer does restart, it will happen earler and without segnificant delay. I'll fix it now